.bash_aliases

```diff
@@ -1,4 +1,8 @@
 # CS 136 DeterLab
+# Check out ssh config files!! They're pretty sweet. I don't know how to do the
+# tunnel with an ssh config file (yet) but I'm 100% sure that there's a way.
+# That being said, this is simpler (I had to special case .local/bin and
+# .ssh/config in my update (bootstrap) script
```

Comment:
I added an SSH config! it also does tunneling :)

Reply:
with what I have now, putting the config in dotfiles/.ssh seems to copy the path into home just fine without a special case!